src/credentials/examples/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -257,6 +257,14 @@ AttestationVerificationResult DefaultDACVerifier::VerifyAttestationInformation(c
     VerifyOrReturnError(GetProductAttestationAuthorityCert(akid, paa) == CHIP_NO_ERROR,
                         AttestationVerificationResult::kPaaNotFound);
 
+    VerifyOrReturnError(IsCertificateTimestampValid(CertificateValidityType::kNotBefore, dacCertDerBuffer, paiCertDerBuffer) == CHIP_NO_ERROR &&
+                            IsCertificateTimestampValid(CertificateValidityType::kNotAfter, dacCertDerBuffer, paiCertDerBuffer) == CHIP_NO_ERROR,
+                        AttestationVerificationResult::kPaiExpired);
+
+    VerifyOrReturnError(IsCertificateTimestampValid(CertificateValidityType::kNotBefore, dacCertDerBuffer, paa) == CHIP_NO_ERROR &&
```

Comment:
It's confusing to have `paa` alone but `dacCertDerBuffer`.

Suggest rename:
- `dacCertDerBuffer` --> `dacDerBuffer`
- `paiCertDerBufer` --> `paiDerBuffer`
- `paa` --> `paaDerBuffer`

Reply:
Fixed in 4ef7efd